README.md

```diff
@@ -208,7 +247,20 @@ To get a list of all supported TLDs, call `supportedTlds`.
 ```php
 $rdapDns->supportedTlds(); // returns an array with all supported TLDs
 ```
+### Working with IPv4/IPv6 registries
 
+Similar to RdapDns you may call getAllIPServers() to see all registries.
+```php
+use Spatie\Rdap\RdapIpV4;
+$ipv4 = new RdapIpV4();
+$ipv4->getAllIPServers(); // returns array<int, array<int, string>>
+```
+To find the server for an IP you may call getServerForIP().
+```php
+use Spatie\Rdap\RdapIpV4;
+$ipv4 = new RdapIpV4();
+$ipv4->getServerForIP("216.58.207.206"); // returns "https://rdap.arin.net/registry/"
```

Comment:
Could you change the casing of `IP` in all functions to `Ip`? That will better match with classes like `RdapIpV4`